src/runtime/audience-action-flow.js

```diff
@@ -91,17 +91,25 @@ export class AudienceActionFlow {
     /** @private @const {?./client-config-manager.ClientConfigManager} */
     this.clientConfigManager_ = deps.clientConfigManager();
 
-    /** @private @const {!ActivityIframeView} */
-    this.activityIframeView_ = new ActivityIframeView(
-      deps.win(),
-      deps.activities(),
-      this.getUrl_(deps.pageConfig(), deps.win()),
-      feArgs({
-        'supportsEventManager': true,
-        'productType': this.productType_,
-      }),
-      /* shouldFadeBody */ true
-    );
+    /** @private @const {!Promise<?ActivityIframeView>} */
+    this.activityIframeViewPromise_ = deps
+      .storage()
+      .get(Constants.USER_TOKEN, true)
```

Comment:
I think you can actually just pass it into the ctor. we have the entitlements response when we create this flow

Reply:
Looks like the entitlements object here doesn't contain the user token.